bot/cogs/ext/commands/theotown.py

```diff
@@ -241,12 +246,21 @@ async def downloadplugin(self, ctx: Context, *, plugin_id: str = None):
         category=TheoTownCategory
     )
     @commands.bot_has_permissions(send_messages=True, attach_files=True, add_reactions=True) # type: ignore # permissions kind of obsolete
-    @command_checks.is_bot_commands()
-    @command_checks.is_theotown_guild()
     @commands.cooldown(rate=1, per=60 * 5, type=commands.BucketType.user) # type: ignore
-    @command_checks.client_is_pidroid()
+    @command_checks.guild_configuration_exists()
     async def suggest(self, ctx: Context, *, suggestion: str = None): # noqa C901
         async with ctx.typing():
+            is_theotown = is_guild_theotown(ctx.guild)
+            if is_theotown:
+                if not is_channel_bot_commands(ctx.channel):
+                    raise InvalidChannel(f'The command can only be used inside <#{BOT_COMMANDS_CHANNEL}> channel')
```

Comment:
I'm not satisfied with this check, there is probably a better way to do this

Reply:
Should probably shift the placement of typing status to the part where it sends a message and starts modifying it. That'd at least be another fix.